Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/Controllers/AppController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
Would add this to existing configuration controller to save an additional http request.

Reply:
Done